.github/workflows/ci.yml

```diff
@@ -3,8 +3,7 @@ name: CI
 on:
   push:
     branches: [main]
-  # eslint-disable-next-line yml/no-empty-mapping-value
```

Comment:
I thought we made this change already in some other PR? we can revert here

Reply:
It's like this on `main`. I copied `{}` instead of `eslint-disable-next-line` from the docs, maybe that's why it  look familiar.